test/unit/helpers/util/Expect.js

```diff
@@ -152,9 +152,10 @@ define([
     });
   };
 
-  fn.isTextField = function ($input) {
+  fn.isTextField = function ($input, inputType) {
```

Comment:
Maybe it would be better to rename this method, as it's purpose changed?

Reply:
Done. Added another method instead of modification the existing one.